js/app.js

```diff
@@ -279,7 +313,8 @@ TacoMap.prototype.saveMarker = function() {
   // stop the bounce animation
   var key = this._db.ref().child('trucks').push().key;
   var update = {};
-  update['/trucks/' + key] = this.getUserPosition();
+
+  update['/trucks/' + key] = $.extend({type: this._toInsert}, this.getUserPosition());
   this._map._isBusy = true;
```

Comment:
We should use the marker type in the Firebase path; everything under /trucks/ is a truck, /skittles/ is a pile of skittles, etc.  Then we can load/display any subset of icons: https://github.com/DevProgress/more-tacos/blob/master/js/app.js#L157.  We'll probably also want to update the logging to include a type: https://github.com/DevProgress/more-tacos/blob/master/js/app.js#L257